source/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -125,10 +147,12 @@ envoy_cc_library(
         "ssl",
     ],
     visibility = ["//visibility:public"],
-    deps = envoy_select_quiche([
-        "//source/common/filesystem:filesystem_lib",
-        "//source/common/http:utility_lib",
-    ]),
+    deps =
+        envoy_select_quiche([
```

Comment:
Nit: please stick to normal Bazel style and have `deps = envoy_select_quiche(...` with no line break.

Reply:
Done.